entrypoint.sh

```diff
@@ -63,4 +63,10 @@ helm dependency update .
 
 # Using the repo name as the chart name and not the chart folder name
 PACKAGE=$(helm package --app-version $CHART_APP_VERSION --version $CHART_VERSION . | rev | cut -d'/' -f1 | rev)
-curl -u "$CHARTMUSEUM_USER:$CHARTMUSEUM_PASSWORD" "$CHARTMUSEUM_URL" --upload-file "$PACKAGE"
\ No newline at end of file
+if [ "$CHARTMUSEUM_URL" == "https://chartmuseum.devops.bestegg.com/"]; then
```

Comment:
I think this needs to be either  
`if [[ "$CHARTMUSEUM_URL" == "https://chartmuseum.devops.bestegg.com/" ]]; then`  
or  
`if [ "$CHARTMUSEUM_URL" = "https://chartmuseum.devops.bestegg.com/" ]; then`